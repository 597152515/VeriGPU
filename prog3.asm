out abx
out cdx
out 12x
out 34x
outloc 20x
outloc 22x
outloc 20x
outloc 22x
outloc 21x

20x:
short deadx
short beefx
