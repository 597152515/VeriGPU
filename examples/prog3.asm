out 0x1b
out 0x2d
out 0x32
out 0x44
outloc 0x40
outloc 0x44
outloc 0x40
outloc 0x44
halt

0x40:
word 0x0000dead
word 0x0000beef
