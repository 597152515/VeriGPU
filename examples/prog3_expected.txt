out 0 00000000000000000000000000011011 0000001b 27
out 1 00000000000000000000000000101101 0000002d 45
out 2 00000000000000000000000000110010 00000032 50
out 3 00000000000000000000000001000100 00000044 68
out 4 00000000000000001101111010101101 0000dead 57005
out 5 00000000000000001011111011101111 0000beef 48879
out 6 00000000000000001101111010101101 0000dead 57005
out 7 00000000000000001011111011101111 0000beef 48879
out 8 00000000000000000001000100010001 00001111 4369
out 9 00000000000000000010001000100010 00002222 8738
out 10 00000000000000000000000000001111 0000000f 15
out 11 00000000000000000000000000010101 00000015 21